File: src/Client/Logistics.OfficeApp/src/app/core/utilities/date-range.ts
export class DateRange {
  constructor(displayName: string, startDate: Date, endDate: Date) {
    this.displayName = displayName;
    this.startDate = startDate;
    this.endDate = endDate;
  }

  public readonly displayName;
  public readonly startDate;
  public readonly endDate;

  toString(): string {
    return this.displayName;
  }
}

File: src/Client/Logistics.DriverApp/Models/DateRange.cs
namespace Logistics.DriverApp.Models;

public class DateRange
{
    public DateRange(string displayName, DateTime startDate, DateTime endDate)
    {
        DisplayName = displayName;
        StartDate = startDate;
        EndDate = endDate;
    }

    public string DisplayName { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }

    public override string ToString()
    {
        return DisplayName;
    }
}
